.github/workflows/factfinder_decennial_manual_update.yml

```diff
@@ -0,0 +1,32 @@
+name: Population Fact Finder - Build Decennial Census Manual Update
+
+on:
+  workflow_dispatch:
+  push:
```

Comment:
looks like this would upload on every push, and maybe every push on every branch?

Reply:
oh i didn't see this was from ur commit just now to test it nvm

Reply:
Yup sorry. Build looks good though! I'll get rid of this in a moment

Reply:
Fixed and good to go